<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Казино</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="start_layout" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>761</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="register_button">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="text">
      <string>Зарегистрироваться</string>
     </property>
    </widget>
    <widget class="QPushButton" name="enter_button">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>320</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Войти</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="pict1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>651</width>
       <height>761</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>pict1</zorder>
    <zorder>register_button</zorder>
    <zorder>enter_button</zorder>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../Desktop/Fon1.qrc"/>
 </resources>
 <connections/>
</ui>
